content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0. The latest release includes over a dozen updates and new features, and it is the first step toward Sourcegraph becoming a code intelligence platform 
```

Comment:
@lcolston08 really small nitpick - I think this description should have a period

```suggestion
description: Announcing Sourcegraph 4.0. The latest release includes over a dozen updates and new features, and it is the first step toward Sourcegraph becoming a code intelligence platform. 
```